metrics/src/main/java/metrics/HealthCheckServletContextListener.java

```diff
@@ -0,0 +1,21 @@
+package metrics;
+
+import com.codahale.metrics.health.HealthCheckRegistry;
+import com.codahale.metrics.servlets.HealthCheckServlet;
+
+import javax.servlet.annotation.WebListener;
+
+/**
+ * Annotated context listener for wiring up the health check servlet.
+ */
+@WebListener
+public class HealthCheckServletContextListener extends HealthCheckServlet.ContextListener {
```

Comment:
This needs to implement the abstract non-static getter required by the abstract ContextListener Dropwizard class.